models/src/test/scala/coop/rchain/models/rholang/SortTest.scala

```diff
@@ -500,6 +504,77 @@ class ParSortMatcherSpec extends FlatSpec with Matchers {
     result.term should be(sortedParMatch)
   }
 
+  it should "sort Lets based on declarations and then body" in {
+    /*
+      let x /\ y = 5 in { 4 } | let x = 4, y = 5 in { 4 } | let x = true, y = 5 in { 4 } ==>
+      let x = true, y = 5 in { 4 } | let x = 4, y = 5 in { 4 } | let x /\ y = 5 in { 4 }
+     */
+    val parLet = Par(
+      lets = List(
+        Let(
+          declarations = List(
+            Declaration(
+              Connective(ConnAndBody(ConnectiveBody(List(EVar(FreeVar(0)), EVar(FreeVar(1)))))),
+              GInt(5),
+              freeCount = 2
+            )
+          ),
+          continuation = GInt(4),
+          bindCount = 2
+        ),
+        Let(
+          declarations = List(
+            Declaration(EVar(FreeVar(0)), GInt(4), freeCount = 1),
+            Declaration(EVar(FreeVar(1)), GInt(5), freeCount = 1)
+          ),
+          continuation = GInt(4),
+          bindCount = 2
+        ),
+        Let(
+          declarations = List(
+            Declaration(EVar(FreeVar(0)), GBool(true), freeCount = 1),
+            Declaration(EVar(FreeVar(1)), GInt(5), freeCount = 1)
+          ),
+          continuation = GInt(4),
+          bindCount = 2
+        )
+      )
+    )
+    val sortedParLet = Par(
```

Comment:
Please add a `Let` that has a `continuation` other than `GInt(4)` so that we can see that the `continuation` part is actually considered.